cmake/EinsumsConfig.cmake.in

```diff
@@ -0,0 +1,167 @@
+# EinsumsConfig.cmake
+# -------------------
+#
+# Einsums cmake module.
+# This module sets the following variables in your project::
+#
+##   ambit_FOUND - true if ambit and all required components found on the system
```

Comment:
Do we want to search/replace ambit with Ensues in this pass?

Reply:
they should all be commented out, and the whole file isn't installed or functional. I was going to leave them for now to keep the high contrast. some things will never get replaced -- `<project>_LIBRARIES` and the like are discouraged for new projects in favor of targets only.